### PR TITLE
Add faq page and java version check info

### PR DESCRIPTION
**Pull request by MiniDigger**
https://github.com/PaperMC/Paper/pull/9692

### CODE REVIEW

File: config/sidebar.paper.ts [hunk 1]
```diff
@@ -33,7 +33,6 @@ const paper: SidebarsConfig = {
             slug: "/cat/admin/how-to-guides",
           },
           items: [
-            "admin/how-to/basic-troubleshooting",
```
Comment (Warriorrrr) [inline config/sidebar.paper.ts:36]: Should probably not be removed

File: docs/paper/admin/reference/system-properties.md [hunk 2]
```diff
@@ -189,3 +189,8 @@ It also remaps plugin CB calls to remove the version information.
 
 - **default**: `true`
 - **description**: Sets whether to parse comments in YAML files by default.
+
+#### Paper.IgnoreJavaVersion:
+
+- **default**: `false`
+- **description**: Allows you to bypass the java version check. See [this](/paper/misc/faq) for more info.
```
Comment (olijeffers0n) [inline docs/paper/admin/reference/system-properties.md:196]: Direct link with anchor is probably more useful here?

File: docs/paper/admin/misc/faq.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Frequently Asked Questions
+
+## Unsupported Java detected, what do I do?!
+
+Unsupported early-access or internal versions of Java are often missing features, have known issues or be straight up broken. 
+As such, we cannot provide support for servers running such versions.
+You should install a supported version of Java as explained [here](/misc/java-install).
+
+If you still wish to continue, knowing that you are on your own and will receive NO support, you can disable the check a system property, as explained [here](/paper/reference/system-properties#paperignorejavaversion).
+
```
Comment (olijeffers0n) [inline docs/paper/admin/misc/faq.md:14]: Double empty lines

File: docs/paper/admin/misc/faq.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+slug: /misc/faq
+---
+
+# Frequently Asked Questions
+
+## Unsupported Java detected, what do I do?!
+
+Unsupported early-access or internal versions of Java are often missing features, have known issues or be straight up broken.
```
Comment (olijeffers0n) [inline docs/paper/admin/misc/faq.md:9]:
```suggestion
Unsupported, early-access, or internal versions of Java are often missing features, have known issues or be straight up broken. 
```

File: docs/paper/admin/misc/faq.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+# Frequently Asked Questions
+
+## Unsupported Java detected, what do I do?!
+
+Unsupported early-access or internal versions of Java are often missing features, have known issues or be straight up broken. 
+As such, we cannot provide support for servers running such versions.
+You should install a supported version of Java as explained [here](/misc/java-install).
+
+If you still wish to continue, knowing that you are on your own and will receive NO support, you can disable the check a system property, as explained [here](/paper/reference/system-properties#paperignorejavaversion).
```
Comment (olijeffers0n) [inline docs/paper/admin/misc/faq.md:13]:
```suggestion
If you still wish to continue, knowing that you are on your own and will receive NO support, you can disable the check, a system property, as explained [here](/paper/reference/system-properties#paperignorejavaversion).
```

File: docs/paper/admin/reference/system-properties.md [hunk 6]
```diff
@@ -189,3 +189,8 @@ It also remaps plugin CB calls to remove the version information.
 
 - **default**: `true`
 - **description**: Sets whether to parse comments in YAML files by default.
+
+#### Paper.IgnoreJavaVersion:
+
+- **default**: `false`
+- **description**: Allows you to bypass the java version check. See [this](/paper/misc/faq#unsupported-java-detected-what-do-i-do) for more info.
```
Comment (olijeffers0n) [inline docs/paper/admin/reference/system-properties.md:196]:
```suggestion
- **description**: Allows you to bypass the java version check. See [here](/paper/misc/faq#unsupported-java-detected-what-do-i-do) for more info.
```

**Review by olijeffers0n — CHANGES_REQUESTED** [review]

File: docs/paper/admin/reference/system-properties.md [hunk 7]
```diff
@@ -189,3 +189,8 @@ It also remaps plugin CB calls to remove the version information.
 
 - **default**: `true`
 - **description**: Sets whether to parse comments in YAML files by default.
+
+#### Paper.IgnoreJavaVersion:
+
+- **default**: `false`
+- **description**: Allows you to bypass the java version check. See [here](/paper/faq#unsupported-java-detected-what-do-i-do) for more info.
```
Comment (Nacioszeczek) [inline docs/paper/admin/reference/system-properties.md:196]: `Java` would look better than `java`

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**